### PR TITLE
[cordova] Improve the mobilespec pack

### PR DESCRIPTION
**Pull request by zhuyongyong**
- support to pack mobilespec apk without crosswalk
- support to update the tools/mobilespec_4.0 directory automatically
- It's successfull to build mobilespec 3.6 and mobilespec 4.0 package

BUG=https://crosswalk-project.org/jira/browse/XWALK-4553

### CODE REVIEW

File: tools/build/pack_cordova_sample.py [hunk 1]
```diff
@@ -605,33 +617,36 @@ def packMobileSpec_cli(app_name=None):
...
+    webview_orgin = os.path.join(BUILD_PARAMETERS.pkgpacktools, "cordova_plugins", "cordova-plugin-crosswalk-webview")
+    if os.path.exists(webview_orgin):
+        if not doCopy(webview_orgin, webview_target):
+            return False
 
-    cordova_mobilespec = os.path.join(BUILD_ROOT, "cordova-mobile-spec")
-    if not doCopy(os.path.join(BUILD_PARAMETERS.pkgpacktools,
-                               "mobilespec", "cordova-mobile-spec"), cordova_mobilespec):
-        return False
+    cordova_mobilespec_orgin = os.path.join(BUILD_PARAMETERS.pkgpacktools, "mobilespec", "mobilespec_4.0", "cordova-mobile-spec")
```
Comment (Honry) [inline tools/build/pack_cordova_sample.py]: typo: s/orgin/origin

**Comment by Honry** [comment]
@qiuzhong, could you help to verify this update?

**Comment by qiuzhong** [comment]
Verifed. We can pack mobilespec successfully with the new mobilespec directory hierarchy.

**Comment by Honry** [comment]
LGTM

**Pull request merged**